src/components/adminNavbar/AdminNavbar.js

```diff
@@ -0,0 +1,38 @@
+import React , { Component } from 'react';
+import { Navbar, Nav, NavItem, NavDropdown } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import ImageFormatter from "../imageFormatter/ImageFormatter";
+import {notificationIcon} from "../../assets/index.js";
+
+class AdminNavbar extends Component {
+    
+    render() {
+        const dropdownName = "Hi, " + this.props.username;
+        return (
+            <div>
+                <Navbar bg="light" variant="light">
+                    <Navbar.Brand>
+                        <Link to="/admin">JJC Admin Panel</Link>
```

Comment:
Can we add JJC logo here instead of plain text?

Reply:
I think that will be covered in the side navbar.

Reply:
Okay, then this looks good!